exchange/src/exchange/traders/trader_types/algo_trader.hpp

```diff
@@ -26,9 +26,9 @@ class LocalTrader : public GenericTrader {
     {}
 
     // Local (algo .py on disk)
-    explicit LocalTrader(std::string algo_path, double capital) :
+    explicit LocalTrader(std::string algo_path, double capital, bool is_binary_algo) :
```

Comment:
At this point I think I favor giving a LocalTrader a WrapperHandle instead of a ton of arguments - it feels cleaner